Board.cs

```diff
@@ -1,43 +0,0 @@
-﻿using System;
```

Comment:
To prawdopodobnie jest niepotrzebne, lepiej zaciągać to co rzeczywiście jest potrzebne

Reply:
Ale pisałeś, że to IDE dodało to automatycznie, więc lepiej przeanalizować czy te importy są potrzebne